frontend/src/config/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "FILE_SIZE_MAX": 40000
```

Comment:
I think we should mention with a comment that 40000 bytes is the **absolute maximum**, and instead put this value to like 32Kb or something

Reply:
I don't really understand :thinking: You want to put 32kb instead of 40kb?

Reply:
I don't understand your point neither :confused: 
But a comment saying that tjis value is in byte would be good!

Reply:
We can gloss over that part, but I 100% think that it should mention in a commentary that 40k bytes is a **hard** maximum so no error happens on the backend. And I agree that saying it is in byte is a good ide